src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -131,6 +131,19 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     with self.argument_context('ams job cancel') as c:
         c.argument('delete', action='store_true', help='Delete the job being cancelled.')
 
+    with self.argument_context('ams content-key-policy') as c:
+        c.argument('account_name', account_name_arg_type)
+        c.argument('content_key_policy_name', name_arg_type)
+        c.argument('description', help='The content key policy description.')
+        c.argument('clear_key_configuration',
+                   action='store_true',
+                   arg_group='Simple Policy Options',
+                   help='Use clear key configuration.')
```

Comment:
`Use Clear Key configuration, a.k.a AES encryption. It's intended for non-DRM keys.` (taken from SDK doc & ms web docs)